djaoapp/templates/users/_user_sidebar.html

```diff
@@ -0,0 +1,25 @@
+<ul class="nav flex-column hidden-sm hidden-md hidden-lg">
+  {% include "generic_navbar.html" %}
```

Comment:
The `generic_navbar.html` should be included at the bottom (see `base_dashboard.html`). Best is actually to move the `{% endblock sidebar %}` in ` base_dashboard.html` so it is included automatically and correctly.

Reply:
I just moved the template from `users/base.html` to this partial, should I still change this? I'm actually not sure what should be done here.